app.js

```diff
@@ -102,6 +107,8 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
+app.use('/ppe', ppeRouter);
```

Comment:
I don't like the `ppe` in the URL, I think we need not mount the routes of `ppeRouter` on any prefix.
`indexRouter` currently has:
1. /contact-us
2. /suggestion

and the `ppeRouter` has:
1. /ppe/map
2. /ppe/create
3. /ppe/list
4. /ppe/save-subscription
5. /ppe (post endpoint)

I don't see any harm in mounting all these on to `/` instead of `/ppe`.

Reply:
I don't have a problem with that. But I prefer the route names to be self-explanatory. So the following routes would make no sense:
* /create (Create what?)
* /list (List what?)
* / POST (???)

`/map` and `/save-subscription` are cool

Reply:
Lets keep it with the' /ppe' route and not on '/', its cleaner that way.

Reply:
I'm in favor of sticking with the `/ppe` in view of avoiding confusion when we add other model types like User